pkger/models.go

```diff
@@ -1002,14 +1070,14 @@ func (c chart) validBaseProps() []validationErr {
 	var fails []validationErr
 	if c.Width <= 0 {
 		fails = append(fails, validationErr{
-			Field: "width",
+			Field: fieldChartWidth,
 			Msg:   "must be greater than 0",
 		})
 	}
 
 	if c.Height <= 0 {
 		fails = append(fails, validationErr{
-			Field: "height",
+			Field: fieldChartHeight,
```

Comment:
all these changes are to couple the field names of the errors with the ones used during parsing, to avoid drift